pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-data-export</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
+  <version>1.0.1-SNAPSHOT</version>
```

Comment:
this shouldn't be modified now

Reply:
we can leave it for now. There will be issues with tags in jenkins as this is already pushed. Let's do it from next time